src/User.php

```diff
@@ -1185,6 +1198,14 @@ public function post_updateItem($history = true)
                 true
             );
         }
+
+        // Hash user_dn if is updated
```

Comment:
This should be done in `pre_updateInDB` to prevent having to run an extra update process.